### PR TITLE
Updater: add support links and some more info

### PR DESCRIPTION
**Pull request by gwolf2u**

### CODE REVIEW

File: src/com/crdroid/updater/UpdatesActivity.java [hunk 1]
```diff
@@ -135,9 +137,160 @@ public void onReceive(Context context, Intent intent) {
                 getString(R.string.header_android_version, Build.VERSION.RELEASE));
 
         TextView headerBuildDate = (TextView) findViewById(R.id.header_build_date);
-        headerBuildDate.setText(StringGenerator.getDateLocalizedUTC(this,
+        headerBuildDate.setText("Current build date: " + StringGenerator.getDateLocalizedUTC(this,
```
Comment (neobuddy89) [inline src/com/crdroid/updater/UpdatesActivity.java:140]: why hard code?

Comment (gwolf2u) [reply src/com/crdroid/updater/UpdatesActivity.java:140]: Because we don't translate this repo anyway

Comment (neobuddy89) [reply src/com/crdroid/updater/UpdatesActivity.java:140]: we might move it to crdroid settings.

**Comment by gwolf2u** [comment]
Can be done afterwards then

File: src/com/crdroid/updater/UpdatesActivity.java (hunk 1 above)
Comment (gwolf2u) [reply src/com/crdroid/updater/UpdatesActivity.java:140]: Can be done afterwards then

**Comment by gwolf2u** [comment]
@neobuddy89 merge as is for now or planning any changes?

**Pull request merged**